pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/d5542db9-894f-48f5-8954-6bb4f7e4bed8?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [478d65a28755b45c2952cc755a0cc8ae466514a7](https://github.com/cockroachdb/cockroach/commits/478d65a28755b45c2952cc755a0cc8ae466514a7):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of1784499128
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:2717: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[13:09:07] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4261: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4261: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of1784499128
--- FAIL: TestCCLLogic_as_of (4.55s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-48275

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/b3253065-dc87-4641-9905-e3981bd8c3a8?testReportRun=2&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [f54cc8e9f0b58664b1fa6db15272889eee8ead41](https://github.com/cockroachdb/cockroach/commits/f54cc8e9f0b58664b1fa6db15272889eee8ead41):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of4216893083
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[12:44:12] --- done: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4262: 
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4262: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of4216893083
--- FAIL: TestCCLLogic_as_of (4.53s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/c659b442-b9e9-46f8-b328-3d66fdbcf136?testReportRun=2&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [1eb87cd23a305418521623cc8059546c92e5c1be](https://github.com/cockroachdb/cockroach/commits/1eb87cd23a305418521623cc8059546c92e5c1be):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of2057306214
    test_log_scope.go:76: use -show-logs to present logs inline
[13:21:55] setting distsql_workmem='56090B';
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[13:21:57] --- done: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4262: 
        /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4262: /var/lib/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of2057306214
--- FAIL: TestCCLLogic_as_of (4.90s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=2</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/1599ed20-5107-4216-8296-97806ec0c222?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [bd6289d21debf18c3bc0f2889bfe98646c60d1d0](https://github.com/cockroachdb/cockroach/commits/bd6289d21debf18c3bc0f2889bfe98646c60d1d0):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of1948642287
    test_log_scope.go:76: use -show-logs to present logs inline
[13:06:47] setting distsql_workmem='70609B';
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[13:06:49] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4262: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4262: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of1948642287
--- FAIL: TestCCLLogic_as_of (4.58s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/a9cc9fda-621d-47d9-9789-5c7ededce788?testReportRun=3&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [e0e47286b08aadf23058825c7b339f19ce852b71](https://github.com/cockroachdb/cockroach/commits/e0e47286b08aadf23058825c7b339f19ce852b71):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of2879297736
    test_log_scope.go:76: use -show-logs to present logs inline
[10:10:02] setting distsql_workmem='100216B';
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[10:10:03] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4262: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4262: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:629: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of2879297736
--- FAIL: TestCCLLogic_as_of (4.71s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/e6e361e5-57e0-48da-b9d8-3ab11474ea0a?testReportRun=3&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [540a0423c37e471e51012b325ec822c03ccb1067](https://github.com/cockroachdb/cockroach/commits/540a0423c37e471e51012b325ec822c03ccb1067):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of3011024031
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[11:32:48] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4262: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4262: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of3011024031
--- FAIL: TestCCLLogic_as_of (4.55s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=3</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/934c80a4-ad22-435b-af5a-14b64f27a98a?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [a064b119c4e9e5629cec5c787f09dea29b60d511](https://github.com/cockroachdb/cockroach/commits/a064b119c4e9e5629cec5c787f09dea29b60d511):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of96973242
    test_log_scope.go:76: use -show-logs to present logs inline
[10:32:13] setting distsql_workmem='56172B';
    logic.go:2718: 
         
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[10:32:15] --- done: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4264: 
        /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4264: /var/lib/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of96973242
--- FAIL: TestCCLLogic_as_of (4.71s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/f0233a27-679a-4e47-9779-f9f9d716e63e?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [7c6e42837c2e475d15e716dacf4b6eb6e8b09e7c](https://github.com/cockroachdb/cockroach/commits/7c6e42837c2e475d15e716dacf4b6eb6e8b09e7c):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of1284431908
    test_log_scope.go:76: use -show-logs to present logs inline
[10:48:07] setting distsql_workmem='76600B';
    logic.go:2723: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[10:48:08] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4278: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4278: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of1284431908
--- FAIL: TestCCLLogic_as_of (4.69s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/ccl/logictestccl/tests/local/local_test.TestCCLLogic_as_of [failed](https://tanzanite.cluster.engflow.com/invocations/default/c9e2ef8d-70c5-4adf-a124-6ac037a1ffa1?testReportRun=1&testReportShard=2&testReportAttempt=1#targets-Ly9wa2cvY2NsL2xvZ2ljdGVzdGNjbC90ZXN0cy9sb2NhbDpsb2NhbF90ZXN0) on master @ [b53856cc8cebf5fe28a8fca3940f5603b5b2d833](https://github.com/cockroachdb/cockroach/commits/b53856cc8cebf5fe28a8fca3940f5603b5b2d833):


```
=== RUN   TestCCLLogic_as_of
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCCLLogic_as_of3428571974
    test_log_scope.go:76: use -show-logs to present logs inline
    logic.go:2723: 
         
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:113: 
        expected:
        (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
        got:
        pq: role/user "root" does not exist
[11:22:25] --- done: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of with config local: 32 tests, 1 failures
    logic.go:4278: 
        /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: error while processing
    logic.go:4278: /var/lib/engflow/worker/work/0/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/logictestccl/tests/local/local_test_/local_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/logictestccl/testdata/logic_test/as_of:116: too many errors encountered, skipping the rest of the input
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCCLLogic_as_of3428571974
--- FAIL: TestCCLLogic_as_of (4.65s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=2</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #142327 pkg/ccl/logictestccl/tests/local/local_test: TestCCLLogic_as_of failed [C-test-failure O-robot P-2 T-sql-foundations branch-release-25.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCCLLogic_as_of.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/logictestccl/testdata/logic_test/as_of
+++ b/pkg/ccl/logictestccl/testdata/logic_test/as_of
@@ -110,11 +110,17 @@ ROLLBACK
 statement error pq: AS OF SYSTEM TIME specified with READ WRITE mode
 BEGIN READ WRITE
 
+statement ok
+SET autocommit_before_ddl = false
+
 statement error (pq: cannot execute CREATE DATABASE in a read-only transaction|database \"\[1\]\" does not exist)
 BEGIN; CREATE DATABASE IF NOT EXISTS d2
 
 statement ok
 ROLLBACK
+
+statement ok
+RESET autocommit_before_ddl
 
 statement ok
 SET DEFAULT_TRANSACTION_USE_FOLLOWER_READS TO FALSE

